{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "135e9517_5063e4da",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1006192
      },
      "writtenOn": "2025-07-28T19:38:46Z",
      "side": 1,
      "message": "The breakage was coming from a merge to stable-3.11, therefore this fix should target stable-3.11.",
      "revId": "2a4140d0ebbda26c213f03d39c6c617a1c328109",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}